website/content/docs/import/gcpsm.mdx

```diff
@@ -0,0 +1,44 @@
+---
+layout: docs
+page_title: Google Cloud Platform Secret Manager - Secrets Import Source
+description: The Google Cloud Platform Secret Manager source imports secrets from GCP to Vault.
+---
+
+# GCP import source
+
+The GCP source enables users to import secret data from GCP Secret Manager into their Vault instance. To be able
+to import secret data from here into your Vault destination, the following permissions are required:
+
+```shell-session
+"secretmanager.secrets.list",
+"secretmanager.versions.access",
+```
```

Comment:
In the Secrets Sync doc this was put into a `Permissions` section, should we do the same?